layouts/default.vue

```diff
@@ -1,9 +1,7 @@
 <script lang="ts" setup>
 import { MediaPlaylistInjectionKey } from "~/plugins/3.mediaPlayer";
 
-const onError = (error: any) => {
-  console.error(error);
-};
+const onError = (error: any) => error;
```

Comment:
What is the reason for this change? 
With this change the `onError` function only returns the error we get from the `NuxtErrorBoundary`, but we dont do anything with it

Reply:
I made these changes to avoid getting the workflow errors! when I do the `git push`

Reply:
Please revert these changes. They are reported as warning by eslint because we use `console`, but its not an error and this shouldn't stop you from contributing.

@sifferhans Related to https://github.com/bcc-code/bmm-web/issues/33 - we should define a strategy of how to deal with errors of various kinds.

Reply:
Nuxt has a `NuxtErrorBoundary` component that catches errors from child components, which has a `@error` event. We could look into using this for client side errors, but we probably need a separate strategy for API errors

Reply:
@sifferhans See https://github.com/bcc-code/bmm-web/issues/33#issuecomment-1531143942. We should move this discussion, its off topic here.